spec.bs

```diff
@@ -1324,12 +1315,6 @@ maps to an internal [=fenced frame config=] [=struct=].
   [Exposed=Window, Serializable]
   interface FencedFrameConfig {
     constructor(USVString url);
-    
-    readonly attribute FencedFrameConfigSize? containerWidth;
-    readonly attribute FencedFrameConfigSize? containerHeight;
-    readonly attribute FencedFrameConfigSize? contentWidth;
-    readonly attribute FencedFrameConfigSize? contentHeight;
```

Comment:
Is the `FencedFrameConfigSize` typedef being used anywhere at this point? It looks like no, so we should probably remove that as well. (Should also clean this up in the implementation while we're at it.)

Same with `FencedFrameConfigURL` I think.

Reply:
Done.